visualization/app/codeCharta/ui/customConfigs/customConfigList/customConfigItemGroup.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component, Input } from "@angular/core"
+
+@Component({
+	template: require("./customConfigItemGroup.component.html")
```

Comment:
selector is missing; therefore `"cc-custom-config-item-group"`, is indeed no Angular element
```suggestion
    selector: "cc-custom-config-item-group",
	template: require("./customConfigItemGroup.component.html")
```

Reply:
You're totally right!